packages/core/__tests__/core.test.ts

```diff
@@ -37,21 +37,21 @@ describe('@actions/core', () => {
 
   it('exportVariable produces the correct command and sets the env', () => {
     core.exportVariable('my var', 'var val')
-    assertWriteCalls([`::set-env name=my var,::var val${os.EOL}`])
+    assertWriteCalls([`::set-env name=my var::var val${os.EOL}`])
```

Comment:
maybe I missed it but do we have any tests that validate multiple properties (and thus does write the comma)?

Reply:
yep command.test.ts is new and tests several combinations